pkg/controller/clusterconfig/clusterconfig_controller.go

```diff
@@ -108,6 +109,18 @@ func (r *ReconcileClusterConfig) Reconcile(ctx context.Context, request reconcil
 	network.MergeClusterConfig(&operConfig.Spec, clusterConfig.Spec)
 
 	if _, ok := clusterConfig.Annotations[names.NetworkTypeMigrationAnnotation]; ok {
+		// Only allows live migration on SD managed clusters
```

Comment:
nit:
```suggestion
		// Only allows live migration on managed clusters
```
not a fan of using "SD" but I'll leave it to you

Reply:
Ill ignore this comment unless others want me to make changes to this PR, then Ill create my own and change this. I agree that id rather drop the SD too.

Reply:
sgtm